index.html

```diff
@@ -443,19 +443,22 @@ <h5 lang="en">Change language</h5>
   </footer>
   <!-- Piwik -->
   <script type="text/javascript">
-    var _paq = _paq || [];
-    _paq.push(["setDocumentTitle", document.domain + "/" + document.title]);
-    _paq.push(["setCookieDomain", "*.www.opensuse.org"]);
-    _paq.push(["setDomains", ["*.www.opensuse.org"]]);
-    _paq.push(['trackPageView']);
-    _paq.push(['enableLinkTracking']);
-    (function() {
-      var u="//beans.opensuse.org/piwik/";
-      _paq.push(['setTrackerUrl', u+'piwik.php']);
-      _paq.push(['setSiteId', 10]);
-      var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-      g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
-    })();
+    // require user consent before processing data
+_paq.push(['requireConsent']);
+_paq.push(['trackPageview']);
+[...]
```

Comment:
I'm quite sure that `[...]` isn't valid javascript ;-)

Reply:
@cboltz not sure what the [...] does, but it was in the Piwik documentation. Any recommendations?

Reply:
maybe code for Et cetera, code to follow and doesn't need to be there.

Reply:
Yes, probably, and removing the `[...]` is the obvious fix.
(And maybe open a bugreport on the Matomo documentation - they should better use `// [...]` (commented out) to avoid such problems if people copy&paste from the documentation.)

That said: I don't know much about Matomo and therefore I can't really answer if the other code you added/changed will work or not.